sunkit_instruments/iris/utils/iris_utils.py

```diff
@@ -23,7 +23,7 @@
 from sunpy.time import parse_time
 import sunpy.util.config
 from sunpy.util.net import check_download_file
-from ndcube import NDCube
+# from ndcube import NDCube
```

Comment:
If ```NDCube``` isn't used, the import should be deleted entirely.

Reply:
Sure